package.json

```diff
@@ -10,11 +10,14 @@
   ],
   "scripts": {
     "lint": "eslint src",
-    "build:lib": "babel src --out-dir lib",
+    "build:lib": "babel src --out-dir lib && flow-copy-source -v src lib",
     "build:umd": "webpack src/index.js dist/react-flip-move.js --config webpack.config.production.js",
     "build:umd:min": "webpack src/index.js dist/react-flip-move.min.js --config webpack.config.production.min.js",
     "build": "npm run build:lib && npm run build:umd && npm run build:umd:min",
-    "prepublish": "npm run build",
+    "flow": "flow",
+    "flow-coverage": "flow-coverage-report -i 'src/**/*.js' -t html -t json -t text",
+    "prebuild": "npm run lint && flow check && npm run test",
```

Comment:
Neat. I wonder if there's a way to run this before `build`, and to abort if any of the steps fail?

If so, that would be great. Would prevent publishing a build with lint/flow errors or test failures.